woocommerce/woocommerce-admin
# Fix Task List styles

Partially solves https://github.com/woocommerce/woocommerce-admin/issues/3897

* Purple icons in the task list should be `$blue-medium-focus`.
* Snackbar styling and positioning should match Gutenberg.
* Image upload button (add a logo task) should match the button in the Image block (https://cloudup.com/cmauCa__tYQ).

### Screenshots

![Screen Shot 2020-06-16 at 11 19 41 AM](https://user-images.githubusercontent.com/1922453/84716780-370adc00-afc8-11ea-97ab-74bfe63e7e70.png)
![Screen Shot 2020-06-16 at 11 33 48 AM](https://user-images.githubusercontent.com/1922453/84716784-383c0900-afc8-11ea-95c0-0ea64e32c3f0.png)
![Screen Shot 2020-06-16 at 11 34 15 AM](https://user-images.githubusercontent.com/1922453/84716789-396d3600-afc8-11ea-8d9b-6232ac0f062d.png)
![Screen Shot 2020-06-16 at 11 57 46 AM](https://user-images.githubusercontent.com/1922453/84716956-a54f9e80-afc8-11ea-943b-4b548d41ad51.png)


### Detailed test instructions:



1. Enable Task List.
2. Make sure the styling looks OK, including the steps themselves.
2. Confirm items found in the bullets above.
2. Do an action to fire a snackbar confirmation. See the new position is the same as the editor.
3. See links should be default blue.

### Changelog Note:



no changelog note.
